velox/functions/sparksql/Register.cpp

```diff
@@ -165,6 +166,37 @@ inline void registerArrayMinMaxFunctions(const std::string& prefix) {
   registerArrayMinMaxFunctions<Timestamp>(prefix);
   registerArrayMinMaxFunctions<Date>(prefix);
 }
+
+void registerToPrettyStringFunctions(const std::string& prefix) {
+  registerFunction<ToPrettyStringFunction, Varchar, int8_t>(
+      {prefix + "toprettystring"});
+  registerFunction<ToPrettyStringFunction, Varchar, int16_t>(
+      {prefix + "toprettystring"});
+  registerFunction<ToPrettyStringFunction, Varchar, int32_t>(
+      {prefix + "toprettystring"});
+  registerFunction<ToPrettyStringFunction, Varchar, int64_t>(
+      {prefix + "toprettystring"});
+  registerFunction<ToPrettyStringFunction, Varchar, float>(
+      {prefix + "toprettystring"});
+  registerFunction<ToPrettyStringFunction, Varchar, double>(
```

Comment:
Better to use a helper function for these numeric types. E.g., add `registerUnaryNumeric` in `RegistrationHelpers.h`, then use it here.